### PR TITLE
Initial query standardization working group proposals.

### PR DESCRIPTION
**Pull request by manolama**
This is an example for the community to comment on and build from.

### CODE REVIEW

File: working-groups/query-standardization/dsl-survey/promql/overview.yaml
Comment (alolita) [inline working-groups/query-standardization/dsl-survey/promql/overview.yaml:1]: Add link for Borgmon - https://sre.google/sre-book/practical-alerting/#:~:text=Borgmon

File: working-groups/query-standardization/use-cases/common/single_type/metrics/select_group_by.yaml
Comment (alolita) [inline working-groups/query-standardization/use-cases/common/single_type/metrics/select_group_by.yaml:1]: Add link for PromQL- https://prometheus.io/docs/prometheus/latest/querying/basics/

**Review by alolita — COMMENTED** [review]
Please add relevant links for each query language. These links for be useful for the reader.

**Review by alolita — COMMENTED** [review]
Context:
- streaming
Streaming is always real-time 
- online
Online could be real-time or batch
- offline
Offline would be batch only?

**Comment by manolama** [comment]
> online
Online could be real-time or batch
offline
Offline would be batch only?

Just trying to capture the real-time or interactive use case (queries returned in less than a few seconds) vs the run and return format where a job is run and you look at the results minutes, hours or a day later.

**Comment by oe73773** [comment]
Can we add an abbreviation.md or glossary.md with the most commenly used abbrevation?
I think this would help people that come in later to catch up.

**Review by alolita — APPROVED** [review]
lgtm.

**Pull request merged**